### PR TITLE
[installer]: create a resource configuration page

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Description

This is almost an FAQ page with all the gotchas we've found over the years. This should be linked to from the KOTS config (see https://github.com/gitpod-io/gitpod/issues/9661) to reduce the support burden and help users get set up quicker.

This only has issues for the container registry - this may have database/object storage gotchas in future

## Related Issue(s)

Fixes #

## How to test

Read the notes

## Release Notes

```release-note
[installer]: create a resource configuration page
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2431"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2431"></a>

### CODE REVIEW

File: gitpod/docs/self-hosted/latest/resource-configuration.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+When using AWS, you should select an in-cluster registry and use S3 as your in-cluster storage provider. This will use the in-cluster registry as a facade for an S3 bucket where all the images will be stored.
+
+If using a region other than `us-east-1`, you will need to change the endpoint to be `s3.<region>.amazonaws.com`.
+
+### [GCP Container Registry](https://cloud.google.com/container-registry)
+
+> This example uses the format `gcr.io`, which is the global hostnames. This can be swapped with any of the [regional hostnames](https://cloud.google.com/container-registry/docs/pushing-and-pulling) and work in the same way.
+
+Google's container registry authentication works in a slightly esoteric way. The "Container registry URL" will be in a format similar to `gcr.io/<project-name>/gitpod`. In order for the authentication to work, you must also enter the "Container registry server". This will always be your GCR hostname, eg `gcr.io`.
```
Comment (lucasvaltl) [inline gitpod/docs/self-hosted/latest/resource-configuration.md:31]: "Google's container registry authentication works in a slightly esoteric way." I don't think we can say that publicly :D Maybe better: "Google's container registry authentication requires specific configuration."

**Comment by lucasvaltl** [comment]
Overall I like this - it makes sense to have this as a guide. Fwiw, I am planning on having a similar but different page in the background section focused on explaining why we need these resources in the first place - you can already find it here: https://github.com/gitpod-io/website/blob/gitpod10126/merge-prod-compat-matrix/gitpod/docs/self-hosted/latest/required-components.md

**Comment by MrSimonEmms** [comment]
> Overall I like this - it makes sense to have this as a guide. Fwiw, I am planning on having a similar but different page in the background section focused on explaining why we need these resources in the first place - you can already find it [here](https://github.com/gitpod-io/website/blob/gitpod10126/merge-prod-compat-matrix/gitpod/docs/self-hosted/latest/required-components.md) and it is part of this [PR](https://github.com/gitpod-io/website/pull/2433)

Excellent. From my quick look, I think they're different enough to justify having two different pages - yours is "why?" and mine is "how?"

**Comment by lucasvaltl** [comment]
Agree - we should link the two once both are merged though :) Just need to coordinate the merging ;D

**Review by lucasvaltl — APPROVED** [review]
LGTM :)

**Pull request merged**